src/js/index.js

```diff
@@ -24,3 +27,37 @@ $(window).scroll(function() {
 });
 
 //Product tile functionality
+
+
+//Sticky nav
+$(document).ready(function(){
+  $(window).resize(function(){
+
+    // Variables
+    var windowHeight = $(window).height();
+    console.log(windowHeight);
+    var toggle = .85 * windowHeight;
+
+    // When the document is scrolled 85%, toggle the classes
+    // Does not work in iOS 7 or below
+    // Hasn't been tested in iOS 8
+    $(document).scroll(function(){
```

Comment:
Scroll event handlers (and resize too) should usually be throttled or debounced. This is really important for creating smooth interactions that aren't too hard on older computers that don't have awesome hardware, like we do.

https://css-tricks.com/the-difference-between-throttling-and-debouncing/